sourcegraph/zoekt
# peterguy/cross platform mmap

Enable cross-platform compiling by using a psutils clone and constraining OS-dependent features.

To test:

1. Run `zoekt-webserver` according to the README
2. Open http://localhost:6070/metrics in your browser (or hit it with `curl`, whatever).
3. Look in the output for the following metrics and ensure that they are displaying values:
- src_disk_space_total_bytes
- src_disk_space_available_bytes
4. If you are on Linux, look for these additional ones:
- proc_metrics_memory_map_max_limit
- proc_metrics_memory_map_current_count
